src/init.c

```diff
@@ -513,6 +513,13 @@ static void jl_uv_exitcleanup_walk(uv_handle_t *handle, void *arg)
 void jl_write_coverage_data(void);
 void jl_write_malloc_log(void);
 
+static struct uv_shutdown_queue_item *next_shutdown_queue_item(struct uv_shutdown_queue_item *item)
+{
+    struct uv_shutdown_queue_item *rv = item->next;
+    free(item);
+    return rv;
+}
```

Comment:
Is what necessary?  Fixing memory leaks?  The goal of the pull request is to fix some of the "definite" memory leaks in julia.  "Everything in the GC" does not qualify as a "definite" memory leak in the valgrind language, because a reference is still held to it at the end of execution.

(Sorry for the number of "quoted phrases"; my comment didn't really seem clear without them.)